services/slam/builtin/builtin_test.go

```diff
@@ -423,27 +425,27 @@ func setupInjectRobot() *inject.Robot {
 			cam.PropertiesFunc = func(ctx context.Context) (camera.Properties, error) {
 				return camera.Properties{}, nil
 			}
-			return cam, nil
+			deps[camera.Named(sensor)] = cam
 		default:
-			return nil, rdkutils.NewResourceNotFoundError(name)
+			deps[camera.Named(sensor)] = cam
```

Comment:
just to verify, should we still be adding a camera to deps here? it looks like this threw an error before

Reply:
NIT: oh i see, there's a new `case` for a sensor that is supposed to be missing (`gibberish`) - I think it's clearer to just to not add the camera to deps in the default case instead of having a specific case for not adding. i'll defer to slam team on this though since it's their test

Reply:
We will discuss it internally with the team but for now, don't want to hold back Kurt's PR since it is correctly updating the code with the current way we are testing. 

Thanks for bring it to my attention though Maxim!